tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1604,7 +1605,15 @@ static __always_inline void* get_path_str(struct path *path)
     set_buf_off(STRING_BUF_IDX, buf_off);
     return &string_p->buf[buf_off];
 }
-
+static __inline int local_strncmp(const char* str1, const char* str2, int len)
+{
+    for (int i=0; i< len ; i++)
```

Comment:
You are not unrolling the loop - this won't work on kernels < 5.2

Reply:
i removed the entire function- see my last comment